data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/races_of_faerun/rac_abilities.lst

```diff
@@ -64,6 +64,15 @@ SOURCELONG:Races of Faerun	SOURCESHORT:Rac	SOURCEDATE:2004-03
 #- Favored Class: Barbarian.
 
 
+Mountain Orc Racial Traits
```

Comment:
Arg, it was meant to be single line, missed that one.
I edit in multiline, but have vim set to convert back to single line when I
write the file, this was unintentional.

On 15 July 2016 at 19:43, Bahamut <notifications@github.com> wrote:

> In
> data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/races_of_faerun/rac_abilities.lst
> <https://github.com/BahamutDragon/pcgen/pull/555#discussion_r71022059>:
>
> > @@ -64,6 +64,15 @@ SOURCELONG:Races of Faerun	SOURCESHORT:Rac	SOURCEDATE:2004-03
> >  #- Favored Class: Barbarian.
> >
> >
> > +Mountain Orc Racial Traits
>
> Multiline is not a default setting in pcgen. For that reason, I only
> accepted single line submissions.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/BahamutDragon/pcgen/pull/555/files/4bdd1a770b75b6fda673507394aad1cddc8ffeeb#r71022059>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGNF3dYANFC-N0-DorkMg8RbGt_LRcyJks5qV9TNgaJpZM4JNZ5k>
> .
>